internal/server/add_payment.go

```diff
@@ -18,13 +18,19 @@ type addPaymentData struct {
 	Success   bool
 	Error     sirius.ValidationError
 
-	Case           sirius.Case
-	Amount         string
-	Source         string
-	PaymentDate    sirius.DateString
-	PaymentSources []sirius.RefDataItem
+	Case              sirius.Case
+	Amount            string
+	Source            string
+	PaymentEvidence   string
+	FeeReductionType  string
+	PaymentDate       sirius.DateString
+	AppliedDate       sirius.DateString
+	PaymentSources    []sirius.RefDataItem
+	FeeReductionTypes []sirius.RefDataItem
```

Comment:
I think it would be best to keep payments and fee reductions separate to keep each handler as simple as possible. We're also going to be adding refunds as well in the near future so if we combine everything in this handler its going to get a little messy and hard to read

Reply:
Happy to create new handlers i agree it could get messy quite fast